packages/backend/src/apis/requestResponses.ts

```diff
@@ -0,0 +1,34 @@
+import axios, { AxiosResponse } from 'axios'
+import { ExtensionList } from '../services/transaction-request-service'
+
+const putBase: string = process.env.PUT_BASE_URI || 'http://localhost:8008' // base uri for testing
+
+export type TransactionRequestResponse = {
+  transactionId?: string;
+  transactionRequestState: 'RECEIVED' | 'PENDING' | 'ACCEPTED' | 'REJECTED';
+  extensionList?: ExtensionList;
+}
+
+export type TransactionRequestError = {
+  errorInformation: {
+    errorCode: string;
+    errorDescription: string;
+    extensionList: ExtensionList;
+  }
+}
+
+export interface RequestResponses {
```

Comment:
Very generic name when our codebase could have lots of different business concerns. Also looks very much like a service. Rather put in /services directory and call `MojaResponseService`.